modules/tensor_mechanics/test/tests/czm/tests

```diff
@@ -13,12 +13,14 @@
     recover = false
     method = '!dbg' # issue 12033
     design = 'source/materials/czm/SalehaniIrani3DCTraction.md source/interfacekernels/CZMInterfaceKernel.md'
+    abs_zero = 1e-4
+    rel_err = 1e-3
```

Comment:
@arovinelli Thanks for addressing this issue.

I'm a little surprised you had to loosen these up this much.  The immediate urgency of getting the failing tests passing is gone, but would you mind taking another look at these to see if you can get the models to converge more tightly so you don't have to use as loose of tolerances in the testing?  

I typically will find the lower bound on how low I can get the residual, and then back the tolerance up a couple of orders of magnitude, and then update the gold file with that more tightly converged solution.  Usually that will make the tests pass on any platform with the default tolerances.

Reply:
@bwspenc I did what you said, so the NL residual saturates around 1e-9 except for 1D case, which saturates around 1e-8. given this I used  `nl_abs_tol=1e-8` and `nl_abs_tol=1e-7`, respectively. I regolded the files and found that for all tests, I can use up to `abs_zero = 1e-6` except for the `czm_3DC_load_complex` for which I needed to use a `rel_err = 1e-4` to make it pass in parallel.

The typical error that we are receiving is 
```
test:czm.czm_3DC_load_Normal: Element variables:
test:czm.czm_3DC_load_Normal:    stress_yy  rel diff:  1.0253557e-07 ~  1.0562696e-07 = 2.92670e-02 (block 2, elmt 8)
test:czm.czm_3DC_load_Normal:    stress_zz  rel diff:  1.0461844e-07 ~  1.0437454e-07 = 2.33137e-03 (block 2, elmt 2)
test:czm.czm_3DC_load_Normal:   --------- Time step 11, 2.0000000e+00 ~ 2.0000000e+00, rel diff:  0.00000e+00 ---------
test:czm.czm_3DC_load_Normal: Global variables:
```
which is a high rel error close to  zero value.

You can see the changes I did [here](https://github.com/idaholab/moose/compare/next...arovinelli:fix_CZM_parallel_tests). Let me know if this is enough or if you want to dig into it more.